wallet/src/ui/app/redux/slices/txresults/index.ts

```diff
@@ -145,18 +145,15 @@ export const getTransactionsByAddress = createAsyncThunk<
                             };
                         })
                         // Remove failed transactions and sort by sequence number
-                        .filter((itm) => itm)
-                        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-                        .sort((a, b) => b!.seq - a!.seq)
+                        .filter(notEmpty)
+                        .sort((a, b) => b.seq - a.seq)
                 );
             });
 
         // Get all objectId and batch fetch objects for transactions with objectIds
         // remove duplicates
         const objectIDs = [
-            ...new Set(
-                resp.filter((itm) => itm).map((itm) => itm?.objectId || '')
```

Comment:
The `''` caused a bunch of console error since it's not a valid object ID. We should probably check this at the TypeScript SDK level.